library/Terminal42/ChangeLanguage/Tests/Helper/LanguageText/LanguageTextWithMapEntriesTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * changelanguage Extension for Contao Open Source CMS
+ *
+ * @copyright  Copyright (c) 2008-2017, terminal42 gmbh
+ * @author     terminal42 gmbh <info@terminal42.ch>
```

Comment:
The copyright for this file is cts, not terminal42 since this is a completely new file. 
Also they are not author and I would suppose they don't want / expect to stand there. 

But If they want it that way it, it would be fine for me

Reply:
de96710